src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  return function(firstNum) {
+    return function(secondNum) {
```

Comment:
```suggestion
  return (firstNum) => {
    return (secondNum) => {
```
You can use arrow functions)